### ISSUE TITLE
{Linked:Bug29617258} Screen reader is not announcing the complete information when focus lands on the expected color value text.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/monaco-editor#5473]
### GitHubTags:
#A11yTCS; #A11ySev3; #A11yMAS;#Desktopweb; #Win11; #Visual Studio Code Client; #VisualStudioCode-Web-Apr2021; #Chrome; #a11y-monacoeditor;#PartnerBug;#WCAG1.3.1;#;#Info and Relationships;

### Environment Details:
Application name: Monaco Editor
Chrome: Version 131.0.6778.205 (Official Build) (64-bit)
Windows 11 Enterprise (Version 23H2)  
OS Build 22631.3737

### Repro Steps:

1. Open URL: https://make.test.powerapps.com/ in edge browser and login with v-id
2. Navigate to 'Apps' control on the left pane and select it.
3. Navigate to 'My apps' table list and select any already created canvas app like "Test123".
4. Navigate all controls under selected "Canvas" app page and navigate to "Copilot functionality menu" on formula bar.
5. Navigate all controls under "Copilot functionality menu" and navigate to the 'Create a formula' menu item and expand it.
6. Now the create a formula dialog will open.
7. Navigate to the edit field and enter the input and activate the apply button.
8. Now the error message is appearing inside the formula bar edit and navigate to the error message.
9. Verify that, Screen reader is announcing the complete information when focus lands on the expected color value text or not.

### Actual Results:
Screen reader is not announcing the complete information when focus lands on the expected color value text

### Expected Results:
Screen reader should announce the complete information when focus lands on the expected color value text
Ex: here it should announce the information as 'Expected color value tooltipColor.white 2+3=5 error at 1:18''.

### User Impact:
Screen reader will not get complete information about the color value text.

### Attachment:

<img width="1919" height="963" alt="Image" src="https://github.com/user-attachments/assets/476f24d5-d7af-4fff-837c-69200d265f77" />
